templates/protostar/error.php

```
@@ -68,6 +68,9 @@
 		</style>
 	<?php endif; ?>
 	<link href="<?php echo $this->baseurl; ?>/templates/<?php echo $this->template; ?>/css/template.css" rel="stylesheet" />
+	<?php if (@filesize('templates/' . $this->template . '/css/user.css') > 5): ?>
```

```suggestion
	<?php if (file_exists('templates/' . $this->template . '/css/user.css')) : ?>
```
Why not use file_exists?

---

can do if you want. I do it this way for my clients as they often add a user.css file and then remove all the content but dont delete the file.

---

I see...so only load it if there is content?

---

Exactly

---

Please use file_exists, suppressing the error is really bad and shouldn't be used in core if possible.

---

It is not suppressing an error at all

---

Thats what @ does

---

The other pages except for the error page link to this file even when empty. Thus it is not consistent on every page.